common/src/main/java/com/wynntils/features/user/FilterAdminCommandsFeature.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.brigadier.tree.CommandNode;
+import com.mojang.brigadier.tree.RootCommandNode;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.CommandsPacketEvent;
+import java.util.Arrays;
+import java.util.List;
+import net.minecraft.commands.SharedSuggestionProvider;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class FilterAdminCommandsFeature extends UserFeature {
+    private static final List<String> FILTERED_COMMANDS = Arrays.asList(
+            "bungee",
+            "connect",
+            "galert",
+            "gcountdown",
+            "glist",
+            "gsend",
+            "perms",
+            "pfind",
+            "plist",
+            "pwlist",
+            "sendtoall",
+            "servers",
+            "sparkb",
+            "sparkbungee",
+            "wcl",
+            "wynnproxy");
+
+    @SubscribeEvent(priority = EventPriority.HIGH)
+    public void onCommandPacket(CommandsPacketEvent event) {
+        RootCommandNode<SharedSuggestionProvider> root = event.getRoot();
+
+        RootCommandNode<SharedSuggestionProvider> newRoot = new RootCommandNode<>();
+        for (CommandNode<SharedSuggestionProvider> child : root.getChildren()) {
+            if (!FILTERED_COMMANDS.contains(child.getName())) {
```

Comment:
Wouldn't a set be more performant?

Reply:
Maybe. I doubt you can even measure it.

The Rules of Optimizations:
1) Don't Optimize
2) (Only for experts) Don't Optimize Yet.

:-)

But I can agree that it makes things a bit clearer (and that's worth more than premature optimization) if you call things a Set when you just need Set-like behavior.

Reply:
... only there is no `asSet`. :( At least not that I can find.

Reply:
> ... only there is no `asSet`. :( At least not that I can find.

`new HashSet<String>(Arrays.asList(...))` is the best I could find 👀